packages/bench/src/types.d.ts

```diff
@@ -16,15 +27,15 @@ export interface BenchSuite {
   // Multiple rolldown options will result in multiple runs with different options.
   // This is useful for benchmarking different options with the same input in rolldown.
   rolldownOptions?:
-    | RolldownOptions
-    | { name: string; options: RolldownOptions }[];
-  rollupOptions?: RollupOptions;
+    | RolldownOptionsWithSingleOutput
```

Comment:
I don't know the context, but it seems to increase the complexity with any benefit. Could you explain why we need to do this?